### DISCUSSION TITLE
Is argobot maintained actively?

### DISCUSSION THREAD

**Discussion by nidhirpatil** [argoproj/argo-cd#14502] [Q&A]
Hi!
I wanted to find out if argocd bot is actively maintained by argoproj labs?
I am looking to integrate it with github to have argo diff and sync available through PRs.
Also, if there are any other approaches to achieve the same, would like to know 🙏

**Comment by jannfis** [comment]
Looking at that repository, the last activity was 3 years ago. So I doubt it that it is actively maintained.
